### DISCUSSION TITLE
Make shift + [ = 「 and shift + ] = 」 in Kana mode?

### DISCUSSION THREAD

**Discussion by HubKing** [google/mozc#909] [Q&A]
Under Microsoft Windows, shift + [ = 「 and shift + ] = 」(US keyboard, Kana mode).  Under Linux (Arch, fcitx5, fcitx5-mozc), shift + [ = ゛for some reason, and Shift + ] = 「 and shift + \ = 」.

Is there an easy way to make  shift + [ = 「 and shift + ] = 」under Linux? I mean, without editing the source code and compiling mozc myself or something.

**Comment by hiroyuki-komatsu** [comment]
Hi HubKing,

You might want to customize romaji table as discussed in previous threads.
* https://github.com/google/mozc/issues/427
* https://github.com/google/mozc/issues/757

Thanks,
